mock/py/mockbuild/package_manager.py

```diff
@@ -293,7 +295,32 @@ def initialize(self):
         self.copy_certs()
         if self.buildroot.is_bootstrap:
             self.copy_distribution_gpg_keys()
-        self.initialize_config()
+        config_content = self.initialize_config()
```

Comment:
nit, instead of returning the variable - could we e.g. set `self.pkg_manager_config` variable and use it?  (it's somewhat weird that the method sets some attributes, and some are returned)